backend/src/main/java/org/cryptomator/hub/validation/ValidPseudoBase64.java

```diff
@@ -0,0 +1,28 @@
+package org.cryptomator.hub.validation;
+
+
+import javax.validation.Constraint;
+import javax.validation.Payload;
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Pattern;
+import java.lang.annotation.Documented;
+import java.lang.annotation.Retention;
+import java.lang.annotation.Target;
+
+import static java.lang.annotation.ElementType.*;
+import static java.lang.annotation.RetentionPolicy.RUNTIME;
+
+@Pattern(regexp = "[+/A-Za-z0-9]+=*")
+@NotNull
+@Target({METHOD, FIELD, ANNOTATION_TYPE, TYPE_USE, PARAMETER})
+@Retention(RUNTIME)
+@Constraint(validatedBy = {})
+@Documented
+public @interface ValidPseudoBase64 {
```

Comment:
I guess by `Pseudo` you're referring to the fact that this check doesn't check whether decoding is actually possible?

Reply:
Then maybe rename it to `OnlyBase64Chars`

Reply:
Fixed in https://github.com/cryptomator/hub/pull/154/commits/90d2dbc60add1185cbfa1ee4be7a2acba01fd50f